app/livechat/client/views/app/livechatInstallation.html

```diff
@@ -1,10 +1,18 @@
 <template name="livechatInstallation">
 	{{#requiresPermission 'view-livechat-manager'}}
-		<p>{{{_ "To_install_RocketChat_Livechat_in_your_website_copy_paste_this_code_above_the_last_body_tag_on_your_site"}}}</p>
+		<p>{{{_ "To_install_the_new_version_of_RocketChat_Livechat_in_your_website_copy_paste_this_code_above_the_last_body_tag_on_your_site"}}}</p>
 
 		<div class="livechat-code">
 			<textarea class="clipboard" data-clipboard-target=".livechat-code textarea">{{script}}</textarea>
 			<button class="button clipboard" data-clipboard-target=".livechat-code textarea"><i class="icon-docs"></i>{{_ "Copy_to_clipboard"}}</button>
 		</div>
+
+		<p>{{{_ "To_install_RocketChat_Livechat_in_your_website_copy_paste_this_code_above_the_last_body_tag_on_your_site"}}}</p>
+
+		<div class="old-livechat-code">
```

Comment:
```suggestion
		<div class="livechat-code clipboard-old">
```

this way you can use the `livechat-code` class for styling, and the new one `clipboard-old` for the clipboard selector